test/functions/operatorOverloads/operatorMethods/secondaryInNonVisibleMod-unrelatedType.chpl

```diff
@@ -1,34 +0,0 @@
-module MyEquals {
-  proc myEquals(x, y) {
-    return x == y;
-  }
-}
-
-
-module MyType {
-  class C {
-    var x: int;
-  }
-
-  operator C.==(a: int, b: int) {
-    // I don't actually expect to see this message.  It is to distinguish from
-    // the default operator and ensure we don't see it.
-    writeln("In C.==(int, int)");
-    return a == b;
-  }
-}
-
-module Main {
-  proc main() {
-    use MyType, MyEquals;
-
-    var myC1 = new C(1);
-    var myC1a = new C(1);
-    var myC2 = new C(2);
-
-    writeln(1 == 1); // expect true
-    writeln(1 == 2);  // expect false
-    writeln(myEquals(1, 1)); // expect true
-    writeln(myEquals(1, 2));  // expect false
-  }
-}
```

Comment:
I'm not familiar with this test, but also not seeing why it's been removed here.  For some reason, I'm not quickly finding my way to the commit that removed it to see if the commit message would say—bringing up the file and its history from here only shows the commit that added it...

Reply:
I removed this test due to a discussion with @lydia-duncan in which the conclusion was that this test didn't make sense as-is and it would be better to have different tests. This PR adds new tests `test/functions/operatorOverloads/shadowing-*.chpl` for this case & I opened up issue #20540 to discuss what we want to have happen here.